### PR TITLE
[PROT-3164][PROT-3170] Enable severity

### PR DESCRIPTION
**Pull request by knizhegorodov**
https://blacklane.atlassian.net/browse/PROT-3164

### CODE REVIEW

**Comment by knizhegorodov** [comment]
@anruban please ping me if you have any experience with fixing those gemspec related build errors

File: README.md [hunk 1]
```diff
@@ -439,6 +439,26 @@ get "/" do
...
+You can specify log level.
+
+```ruby
+Kiev.configure do |config|
+  # One of: 0, 1, 2, 3, 4 (DEBUG, INFO, WARN, ERROR, FATAL)
+  config.log_level = 0
+end
+```
+
+### enable_filter_for_log_levels
```
Comment (anruban) [inline README.md:452]: Let's add a piece of information about a default behavior, WDYT?
To reflect that if no `enable_filter_for_log_levels` set then the filtering will be applied to all levels.

File: spec/lib/kiev_spec.rb [hunk 2]
```diff
@@ -62,5 +62,66 @@
           .with(data, Kiev::Config.instance.filtered_params, Kiev::Config.instance.ignored_params)
       end
     end
+
+    describe "event with predefined severity" do
+      shared_examples "with log severity" do |severity|
+        it "has log severity" do
+          initial = described_class.log_level
+          described_class.configure do |c|
+            c.log_level = ::Logger::DEBUG
```
Comment (anruban) [inline spec/lib/kiev_spec.rb:71]: Honestly, didn't get the idea of temporary setting `::Logger::DEBUG`, what is the reason?

**Review by anruban — COMMENTED** [review]
The implementation overall looks good.

I'd update a [README](https://github.com/blacklane/kiev/blob/master/README.md#other-events) more by new added methods (`.debug`, `.info`, etc).

Also, it seems we broke the documented behavior, so the update is required. Calling `Kiev.logger` is not available anymore.
```ruby
Kiev.logger.info("hello world")
Kiev.logger.debug({ first_name: "john", last_name: "smith" })
```
Otherwise, we could save `Kiev.logger` for backward compatibilities. I'd say this approach is better in my opinion.

File: spec/lib/kiev_spec.rb (hunk 2 above)
Comment (knizhegorodov) [reply spec/lib/kiev_spec.rb:71]: Info is default one. For example `debug` method  in specs will produce nothing till we change `log_level `

**Review by anruban — APPROVED** [review]

**Pull request merged**